dynamic_rest/renderers.py

```diff
@@ -0,0 +1,93 @@
+from collections import defaultdict
+from rest_framework import renderers
+
+
+class DynamicJSONRenderer(renderers.JSONRenderer):
+
+  def render(self, data, accepted_media_type=None, renderer_context=None):
+    # if data is a string, just render it out using the default method
+    # this usually indicates an error message
+    if isinstance(data, basestring):
+      return super(DynamicJSONRenderer, self).render(data, accepted_media_type, renderer_context)
+
```

Comment:
This renderer actually does 4 things right now:

1) strips out `_model` and `_pk` tags added by serializers
2) modifies response such that it is keyed by the serializer resource name
3) sideloading
4) adds `meta` into the response (for pagination)

I am thinking that we should always try to do (1) and allow the viewset to control (2) (3) and (4) -- i.e. you should be able to have a response without `meta` (if the viewset has no meta), with embedded rather than sideloaded records (if the viewset disables sideloading, on by default), and without the Ember-ified structure (if the viewset disables it). And really, we can make (1) an option as well for debugging purposes :)